code/game/machinery/vending/vendor_types/squad_prep/squad_tl.dm

```diff
@@ -44,7 +45,6 @@ GLOBAL_LIST_INIT(cm_vending_gear_tl, list(
 		list("Machete Pouch (Full)", 15, /obj/item/storage/pouch/machete/full, null, VENDOR_ITEM_REGULAR),
 		list("Fire Extinguisher (Portable)", 5, /obj/item/tool/extinguisher/mini, null, VENDOR_ITEM_REGULAR),
 		list("Whistle", 5, /obj/item/device/whistle, null, VENDOR_ITEM_REGULAR),
-		list("Welding Goggles", 5, /obj/item/clothing/glasses/welding, null, VENDOR_ITEM_REGULAR),
```

Comment:
Preferably don't remove these just add another option. I wanted to move away from sci-fi eye pieces but welder goggles fit perfectly in the setting aesthetic.